101-XSS-Attack-Prevention/azuredeploy.json

```diff
@@ -0,0 +1,622 @@
+          },
+          "tags": {
+            "value": "[variables('tags')]"
+          }
+        }
+      }
+    },
+    {
+      "apiVersion": "2018-05-01",
+      "name": "[concat(variables('sqlServerName'),'-auditingSettings','-resource')]",
+      "type": "Microsoft.Resources/deployments",
+      "dependsOn": [
+        "[concat(variables('sqlServerName'),'-resource')]",
+        "[concat(variables('diagStorageAccName'),'-resource')]"
+      ],
+      "properties": {
+        "mode": "Incremental",
+        "templateLink": {
+          "uri": "[variables('sqlAuditingTemplateUri')]"
+        },
+        "parameters": {
+          "sqlServerName": {
+            "value": "[variables('sqlServerName')]"
+          },
+          "storageAccountName": {
+            "value": "[variables('diagStorageAccName')]"
+          }
+        }
+      }
+    },
+    {
+      "apiVersion": "2018-05-01",
+      "name": "[concat(variables('sqlServerName'),'-securityAlertPolicies','-resource')]",
+      "type": "Microsoft.Resources/deployments",
+      "dependsOn": [
+        "[concat(variables('sqlServerName'),'-auditingSettings','-resource')]"
+      ],
+      "properties": {
+        "mode": "Incremental",
+        "templateLink": {
+          "uri": "[variables('sqlSecurityTemplateUri')]"
+        },
+        "parameters": {
+          "sqlServerName": {
+            "value": "[variables('sqlServerName')]"
+          },
+          "storageAccountName": {
+            "value": "[variables('diagStorageAccName')]"
+          },
+          "sendAlertsTo": {
+            "value": "[parameters('emailToSendAlertsTo')]"
+          }
+        }
+      }
+    },
+    {
+      "apiVersion": "2018-05-01",
+      "name": "[concat(variables('databases')[0].name,'-database','-resource')]",
+      "type": "Microsoft.Resources/deployments",
+      "dependsOn": [
+        "[concat(variables('sqlServerName'),'-resource')]"
+      ],
+      "properties": {
+        "mode": "Incremental",
+        "templateLink": {
+          "uri": "[variables('sqlDatabaseTemplateUri')]"
+        },
+        "parameters": {
+          "sqlServerName": {
+            "value": "[variables('sqlServerName')]"
+          },
+          "location": {
+            "value": "[parameters('location')]"
+          },
+          "databaseName": {
+            "value": "[variables('databases')[0].name]"
+          },
+          "omsWorkspaceResourceId": {
+            "value": "[reference('deploy-xss-attack-oms-resource').outputs.workspaceId.value]"
+          },
+          "tags": {
+            "value": "[variables('tags')]"
+          },
+          "administratorLogin": {
+            "value": "[parameters('sqlAdministratorName')]"
+          },
+          "administratorLoginPassword": {
+            "value": "[parameters('sqlServerPassword')]"
+          },
+          "bacpacuri": {
+            "value": "https://templatebotstorage.blob.core.windows.net/bacpacs/contosoclinic.bacpac"
```

Comment:
remove the bacpac file from the sample since we're not using it

Reply:
removed bacpac file